docs/context/ref/Contact.md

```diff
@@ -52,6 +52,7 @@ Intents
 - [ViewResearch](../../intents/ref/ViewResearch)
 - [ViewInteractions](../../intents/ref/ViewInteractions)
 - [ViewOrders](../../intents/ref/ViewOrders)
+- [CreateInteraction](../../intents/ref/CreateInteraction)
```

Comment:
alpha sort or rebase PR so this change isn't in here. If you've fixed in the other PR you can merge it into this to bring across - although we prefer clean PRs based off master (rather than branches used in other PRs)